libvast/src/format/mrt.cpp

```diff
@@ -291,13 +290,13 @@ reader::reader(caf::atom_value table_slice_type)
     {"old_state", count_type{}},
     {"new_state", count_type{}},
   }}.name("mrt::bgp4mp::state_change");
+  if (input)
```

Comment:
Is it a problem to move from a non-initialized `unique_ptr`?

Reply:
Not it isn't. I intended to call `reset()` here instead of assigning directly.

Reply:
A default-constructed `unique_ptr` is null according to the standard. So there are no surprises from unitialized pointers.